README.md

```diff
@@ -1 +1,6 @@
+[![npm](https://img.shields.io/npm/v/@nomiclabs/buidler-solhint.svg)](https://www.npmjs.com/package/@nomiclabs/buidler-solhint)
+ [![Build Status](https://travis-ci.com/nomiclabs/buidler-solhint.svg?branch=master)](https://travis-ci.com/nomiclabs/buidler-solhint)
+
+
 # buidler-solhint
```

Comment:
A readme based on [this one](https://github.com/nomiclabs/buidler-web3) would be great.